### PR TITLE
[DCJ-89][risk=no] Convert index.js to index.tsx

### PR DESCRIPTION
**Pull request by rushtong**
### Addresses
Addresses, but does not complete, work done for https://broadworkbench.atlassian.net/browse/DCJ-89

### Summary
This PR breaks out the index js -> tsx conversion from here: https://github.com/DataBiosphere/duos-ui/pull/2512/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import {BrowserRouter} from 'react-router-dom';
+
+const load = async () => {
+    unregister();
+    const container = document.getElementById('root');
+    const root = createRoot(container!);
+    root.render(<BrowserRouter><App/></BrowserRouter>);
+};
+
+load();
```
Comment (fboulnois) [inline src/index.tsx]:
```suggestion
await load();
```

If we're doing this asynchronously, doesn't this need to be await?

**Review by fboulnois — COMMENTED** [review]
See below:

**Review by aarohinadkarni — APPROVED** [review]
looks good! 👍 after Florian's comment is addressed

**Review by fboulnois — APPROVED** [review]
thank you 👍

**Pull request merged**